### DISCUSSION TITLE
Access denied message the 2nd time I ran the testbed.exe

### DISCUSSION THREAD

**Discussion by mhussar** [NVlabs/instant-ngp#378] [Q&A]
Don't know why it worked find the first time and the second time I ran it Windows said I couldn't run this software on the pc.

**Comment by shamirandres** [comment]
Any awnswers on this?

**Comment by zhengFang42** [comment]
same problem here

**Reply by bobzombi** [reply]
Same here, ran it the first time, worked well. Then crashed ad now I cannot run it again.
I either get a this app can't run on your pc. or acces denied. 
I have set the compatibility of textbed.exe to windows 7 or 8 or turned it off. But all the same result.

**Comment by LaslowAlarcon** [comment]
Hi! Had the same issue until I noticed the file (testbed.exe) had a 0 kb size, when originaly is 54.3 mb.
What I suggest is that you reinstall all the contents from the instant-ngp folder and make a backup of the file. 
Found out that closing the .exe with the close window button corrupted the file so use the Esc key instead. 
Hope this helps n.n

PSA: If someone can provide the repository where the Testbed.exe file is, it can be as straight forward as downloading it from there instead of downloading the whole thing again.

**Reply by ogradyso** [reply]
The Testbed.exe file needs to be compiled (the build step) for the specific CPU and GPU architecture that it will be running on. That's why the repo is using CMAKE because it handles a lot of the compilation for you. It would be difficult for the maintainers of this repo to include Testbed.exe files because they would only work for Windows users.

**Comment by ogradyso** [comment]
The access denied message is probably from your system, meaning that the Testbed.exe file tried to open/change/create a file and the OS denied the access. I would suggest removing the original version, clone a new one and see if the problem happens again. 

The good news is you won't have to reinstall all the dependencies and put them on your system paths because if it worked before it means the dependencies were in the right location/path.

**Comment by Slowdance1** [comment]
Hi, was this solved? My testbed.exe is 0B
